samples/Ralltiir/Ralltiir.Server/Models/ApplicationUser.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Identity;
+
+namespace Ralltiir.Server.Models
+{
+    public class ApplicationUser : IdentityUser<Guid>
+    {
+        [Key]
+        public Guid UserId { get; set; }
```

Comment:
Why do we need an extra ID here? Consider removing the two models and using the default Identity models instead.

Reply:
There was a comment on how to change the ID type but no sample actually showed it. Since this sample also deals with roles, I thought it was a good chance to change the IDs to GUIDs since it also shows that you need to use the generic versions of a lot of the mangers.

Reply:
Changing the PK type is fine, but here you're not only changing the type of the `Id` property (set via the generic type parameter), you're also adding a new one so you basically end up with 2 key properties.

Reply:
Oh... my bad. 🤦🏻